### PR TITLE
general style improvements and refactoring

### PR DESCRIPTION
**Pull request by roosdebildt**
General styling more in line with railsgirlssummerofcode.com:
- deletes non-functioning color-stack and creates new one with matching colors
- adds Lato font-stack
- restructures folders and files to better control separate stylings and their order
- removes border-radius of avatars and other images

### CODE REVIEW

File: app/assets/stylesheets/_colors.sass [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// This is the RGSoC teams-app color stack:
+
+$main: 		#333
+$railsred:	#d82222
+$lightgray:	#ddd
+$darkgray: 	#999
+$white: 	#fff
```
Comment (yorickpeterse) [inline app/assets/stylesheets/_colors.sass]: You're using tabs here for alignment, use spaces instead (tabs are not guaranteed to have an exact/equal width).

Comment (roosdebildt) [inline app/assets/stylesheets/_colors.sass]: AAAAAH I keep on changing my indentation setting but they don't seem to stick. googling.it.now.

**Comment by roosdebildt** [comment]
@alicetragedy you might want to keep an eye on the changes here, too. I'm refactoring big time :) more to come!

**Comment by alicetragedy** [comment]
thanks for the heads up :+1: good to know!

**Comment by alicetragedy** [comment]
Looks good to me – I really like the way the Lato font looks on the teams app. :+1:
As already commented, I'm not sure about the callout thing :)
I just noticed some minor things in the calendar (the styling changes a lot) but it actually looks better this way, apart from the "previous/next" arrows which are a bit small – but I think it can always be fixed at a later point.

**Comment by roosdebildt** [comment]
In my search results, I found it was being used in a helper:

app/helpers/application_drafts_helper.rb:

   `15      if application_draft.as_student?`
   `16:       content_tag :div, class: 'bs-callout bs-callout-warning' do`
   `17          capture(&header) + capture(&block)`
   `18        end`

I'm not sure what it is exactly supposed to do though, but I though better not delete it ;) What do you think @alicetragedy?

Ah, and I see what you mean with the calendar... As soon as we will have events I can take care of that!

**Comment by alicetragedy** [comment]
@roosdebildt ah, I see it now. I suggest keeping it the way you did – once I have a little bit more time (today or tomorrow) I will take a look at whether the 'callout' actually works or not. 
Some conferences have been added this weekend, so as of now we officially have events already!

**Comment by roosdebildt** [comment]
@alicetragedy I found the problem! :dancers: the fullcalendar-stylesheet was hidden in a weird folder "vendor" that was only used for that stylesheet. I integrated it in my reorganisation of all folders and files. I did remove the .gitkeep-files too though. 

Also, the callout thing is just a warning-message that pop's up in some part of filling out the application. So I think we just leave it, right?

You think it's ok to merge as it is now? Then I will make a PR to upstream. :smile:

**Comment by carpodaster** [comment]
@roosdebildt You don't have to merge this PR into your fork. In fact, it would be better not to since that will create merge commits which will be part of your PR to the upstream repo.

The best workflow would be to never commit to your fork's master, only to branches. That way you can always to a clean `git pull` from upstream and PR from your fork won't include undesired commits.

**Comment by roosdebildt** [comment]
@carpodaster yes, that is actually exactly what I meant as well! But I thought for the workflow it's better to first make PR's to our fork so we can comment on it and make it "perfect" and once it's shippable merge it with upstream. That's the way to go, right?

**Comment by roosdebildt** [comment]
@F3PiX didn't we say we wanted to close the PR to our fork without merging it? and pull the changes from upstream...? I'm confused! :confused:

**Comment by emcoding** [comment]
We did indeed. And now Upstream is taking care of closing our PR in our fork. Automagically.

**Pull request merged**